contracts/FeralfileArtworkV4_1.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "./FeralfileArtworkV4.sol";
+
+contract FeralfileExhibitionV4_1 is FeralfileExhibitionV4 {
+    struct AdvancedData {
+        address artist;
+        uint256 amount;
+    }
+
+    mapping(address => uint256) public artistAdvancedAmounts;
+
+    constructor(
+        string memory name_,
+        string memory symbol_,
+        bool burnable_,
+        bool bridgeable_,
+        address signer_,
+        address vault_,
+        address costReceiver_,
+        string memory contractURI_,
+        uint256[] memory seriesIds_,
+        uint256[] memory seriesMaxSupplies_,
+        AdvancedData[] memory data_
```

Comment:
I'd suggest passing data with the array of primitive type, i don't think create a struct just serve for the constructor is efficient in term of gas fee and also programming. We could use 2 different arrays like `advanceAddresses` and `advanceAmounts` and parse them by index respectively.